jenkins/tenant/aws/ecoeng_01/PolicyJenkinsfileDaily

```diff
@@ -39,6 +39,9 @@ pipeline {
         contact4 = "matrujil@redhat.com"
         // Find the all available policies: https://github.com/redhat-performance/cloud-governance/tree/main/cloud_governance/policy
         // By default, all policies are running in dry_run="yes" mode and the whole list can be found in run_policies.py
+        // POLICIES_IN_ACTION: Policies that run in the dry_run="no" mode
+        POLICIES_IN_ACTION = '["unattached_volume", "ip_unattached", "zombie_snapshots", "unused_nat_gateway", "s3_inactive", "empty_roles"]'
```

Comment:
Why did u delete "zombie_cluster_resource" ?

Reply:
I want to simulate the old scenario. After adding the zombie_cluster_resources policy it behaving wiered.